doc/source/user_guide/sparse.rst

```diff
@@ -266,7 +266,7 @@ have no replacement.
 
 .. _sparse.scipysparse:
 
-Interaction with scipy.sparse
+Interaction with *scipy.sparse*
 -----------------------------
```

Comment:
Can you also adapt the underline?

Reply:
I'm not sure what you mean here

Reply:
I understand now; I'm meant to extend the underline to be the same length as the text

Reply:
Indeed